CampaignConfig/mdc2020_trkana.ini

```diff
@@ -0,0 +1,42 @@
+[campaign]
```

Comment:
Where are memory and cpu/wall time limits set? TrkAna jobs should be able to run with very small footprints and time limits.

Reply:
Will check